tests/qeqiskit/backend/backend_test.py

```diff
@@ -132,7 +138,7 @@ def test_run_circuitset_and_measure(self, backend):
             # Then (since SPAM error could result in unexpected bitstrings, we make sure
             # the most common bitstring is the one we expect)
             counts = measurements.get_counts()
-            assert max(counts, key=counts.get) == "100"
+            assert max(counts, key=counts.get) == "1"
```

Comment:
Why should it be "1" now if we still are doing 100 samples?

Reply:
it cuts down on the computation time a bit. Beause you are using a simulator there is no risk of SPAM errors. 

Maybe I should remove that comment as it is confusing.